include/remill/Arch/Arch.h

```diff
@@ -59,6 +60,7 @@ class IntegerType;
 class LLVMContext;
 class Module;
 class PointerType;
+class AttributeMask;
```

Comment:
Why do you forward declare it if you include it above? If a forward declaration is sufficient then that is the preferred solution for this header, as `Arch.h` is included in a lot of places.